django/db/models/options.py

```diff
@@ -826,3 +826,12 @@ def _property_names(self):
             if isinstance(attr, property):
                 names.append(name)
         return frozenset(names)
+
+    @cached_property
+    def returning(self):
```

Comment:
I guess this should be something like `returning_fields` -- `returning` looks like a boolean attribute as used elsewhere.